checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```
@@ -0,0 +1,492 @@
+      - User
+    name: User Name
+    path: usr.name
+    source: log
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Application Name
+    path: checkpoint.quantum.firewall.appi_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Application Risk
+    path: checkpoint.quantum.firewall.app_risk_str
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Client Inbound Bytes
+    path: checkpoint.quantum.firewall.client_inbound_bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Client Inbound Packets
+    path: checkpoint.quantum.firewall.client_inbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Client Outbound Bytes
+    path: checkpoint.quantum.firewall.client_outbound_bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Client Outbound Packets
+    path: checkpoint.quantum.firewall.client_outbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Inbound Packets
+    path: checkpoint.quantum.firewall.server_inbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Outbound Packets
+    path: checkpoint.quantum.firewall.server_outbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Total Bytes
+    path: checkpoint.quantum.firewall.bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+pipeline:
+  type: pipeline
+  name: Checkpoint Quantum Firewall
+  enabled: true
+  filter:
+    query: "source:checkpoint-quantum-firewall"
+  processors:
+    - type: service-remapper
+      name: Define `product` as the official service of the log
+      enabled: true
+      sources:
+        - product
+    - type: date-remapper
+      name: Define `time` as the official date of the log
+      enabled: true
+      sources:
+        - time
+    - name: Lookup for `severity` to `severity` field
+      enabled: true
+      source: severity
+      target: severity
+      lookupTable: |-
+        0, Info
+        1, Low
+        2, Medium
+        3, High
+        4, Critical
+        Info, Info
+        Low, Low
+        Medium, Medium
+        High, High
+        Critical, Critical
+      type: lookup-processor
+    - name: Lookup for `severity` to `severity_to_status` field
+      enabled: true
+      source: severity
+      target: severity_to_status
+      lookupTable: |-
+        Info, info
```

nit, does it make sense to differentiate info from low

```suggestion
        Info, debug
```

---

Done